SpokojDucha/components/SearchDecedent.js

```diff
@@ -15,12 +15,6 @@ const SearchDecedent = () => {
   const [selectedImage, setSelectedImage] = useState(null);
   const navigation = useNavigation();
 
-  const formatDate = (dateArray) => {
```

Comment:
Why we suddenly don't have this function? If you delete someone's code, give a reason. I don't see a reason. The code is needed for the handling of displaying data in correct format